hack/make/debug/csi-provisioner.patch

```diff
@@ -0,0 +1,81 @@
+From: A happy user <happy@example.com>
```

Comment:
Couldn't this be done using helm instead + a `debug.Dockerfile`? (I guarantee that someone will commit the changes)

Like have a little `_debug.tpl` that have template functions that will add the same things to the manifests if a flag `debug` is `true`

IMO, these git patches can get easily outdated

Reply:
Also patch won't work as soon as we change something in this file. So better to introduce DEBUG env var or so. or use template.

Reply:
Did it, as we once agreed on that we won't use a second Dockerfile in the repo, but yes, I agree that it would make more sense with a flag. Will change it 👍  - Thx for the input

Reply:
Added a debug flag